datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/tag/TagType.java

```diff
@@ -1,22 +1,47 @@
 package com.linkedin.datahub.graphql.types.tag;
 
+import com.linkedin.common.AuditStamp;
+import com.linkedin.common.Owner;
+import com.linkedin.common.OwnerArray;
+import com.linkedin.common.Ownership;
+import com.linkedin.common.OwnershipSource;
+import com.linkedin.common.OwnershipSourceType;
+import com.linkedin.common.OwnershipType;
+import com.linkedin.common.urn.CorpuserUrn;
 import com.linkedin.common.urn.TagUrn;
+import com.linkedin.data.template.SetMode;
 import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.generated.AutoCompleteResults;
 import com.linkedin.datahub.graphql.generated.EntityType;
+import com.linkedin.datahub.graphql.generated.FacetFilterInput;
+import com.linkedin.datahub.graphql.generated.SearchResults;
 import com.linkedin.datahub.graphql.generated.Tag;
+import com.linkedin.datahub.graphql.generated.TagUpdate;
+import com.linkedin.datahub.graphql.resolvers.ResolverUtils;
+import com.linkedin.datahub.graphql.types.MutableType;
+import com.linkedin.datahub.graphql.types.mappers.AutoCompleteResultsMapper;
+import com.linkedin.datahub.graphql.types.mappers.SearchResultsMapper;
 import com.linkedin.datahub.graphql.types.tag.mappers.TagMapper;
+import com.linkedin.datahub.graphql.types.tag.mappers.TagUpdateMapper;
+import com.linkedin.metadata.configs.TagSearchConfig;
+import com.linkedin.metadata.query.AutoCompleteResult;
+import com.linkedin.r2.RemoteInvocationException;
+import com.linkedin.restli.common.CollectionResponse;
 import com.linkedin.tag.client.Tags;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 import java.net.URISyntaxException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class TagType implements com.linkedin.datahub.graphql.types.EntityType<Tag> {
+public class TagType implements com.linkedin.datahub.graphql.types.SearchableEntityType<Tag>, MutableType<TagUpdate> {
 
     private static final String DEFAULT_AUTO_COMPLETE_FIELD = "name";
+    private static final TagSearchConfig TAG_SEARCH_CONFIG = new TagSearchConfig();
```

Comment:
No other instance we can reference?

Reply:
Not sure- this was copy+pasted from ChartType and DashboardType